docs/active-directory/core-concepts/01-active-directory-details-and-definitions.md

```diff
@@ -0,0 +1,111 @@
+# Active Directory Details and Definitions 
+
+### Active Directory Domain Services
+
+Active Directory Domain Services (AD DS, or more simply just AD for Active Directory) is a storage and management
+mechanism for everything related to a collection of resources within a computer network. This includes users, files,
+network configurations, and devices like physical computers and printers. Administrators can modify the Active Directory
+and remotely manage all the components currently tracked at once. In its simplest terms, **AD is used to logically
+represent an organizations resources and users and enforce policies and behaviors across them.**
+
+AD Includes the following features
+
+- Data schemas which enforce particular requirements onto AD data
+- A global catalog of all data
+- Query and Indexing mechanisms for finding and publishing specific data
+- Database replication to various Domain Controllers (DC)
+
+### AD Storage Structure
+
+[From MS Docs](https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc759186(v=ws.10))
+
+“*Administrators use **Active Directory** to store and organize objects on a network (such as users, computers,
+devices, and so on) into a secure hierarchical containment structure that is known as the **logical structure**.
+Although the **logical structure** of **Active Directory** is a hierarchical organization of all users, computers,
+and other physical resources, the **forest** and **domain** form the basis of the logical structure. **Forests**,
+which are the **security boundaries of the logical structure**, can be structured to provide data and service autonomy
+and isolation in an organization in ways that can both reflect site and group identities and remove dependencies on the
+physical topology.”*
```

Comment:
Use a block quote, i.e.

> This is a block quote

Reply:
Also, if they wanted to read this, couldn't someone just click on the MS docs linked? In this section, I think it's better to link to the docs and instead split this up into two subsections:
1. A terminology section that just covers each term and a brief description
2. A "dumbed down" version of the docs, leveraging an analogy that explains the relationship between all of the above terms

i.e.

---

Since Microsoft assumes that an organizational's entire view of all of the resources owned by it is tracked within the Active Directory, it supports the ability to form logical groupings of those resources (i.e. **organizational units**) within the Active Directory that represent discrete segments of your organization. These discrete segments can be managed independently by administrators, which can be used to apply security and organizational policies to groups of resources as a whole.

As an analogy, if you think of Active Directory as a supermarket, **forests** represent different divisions of the supermarket (i.e. Home, Technology, Grocery, etc.) whereas **domains** are like aisles within a specific division (i.e. you may have a aisle within Grocery dedicated to breakfast items).

Active Directory represents these entities and their relationships using a flat database file that encodes a tree of relationships. For example, in our supermarket analogy:

```
Grocery/
  Breakfast Foods/
  Baking Goods/
  etc.
Home/
  etc.
Technology/
  etc.
```

---

Once you have such an analogy as above to give a strong image of what we're talking about, the more complicated description becomes more accessible to read!

Concepts like the fact that multiple forests exist in a single ActiveDirectory also more naturally flow from that analogy.